web/client/components/notifications/NotificationContainer.jsx

```diff
@@ -78,10 +83,25 @@ class NotificationContainer extends React.Component {
         });
         notifications.forEach(notification => {
             if (systemNotifications.indexOf(notification.uid) < 0) {
+                let id = notification.message;
+                let str = LocaleUtils.getMessageById(this.context.messages, id);
```

Comment:
I would add formatting support with values directly in LocaleUtils.getMessageById(context, id, values)

Reply:
you cannot do that with react-int api, because they are injected from an HOC and can't be used in the LocaleUtils.
for what you are suggesting we should look at [Intl MessageFormat](https://github.com/yahoo/intl-messageformat)

Reply:
ok, for now